src/Psy/CodeCleaner/ValidClassNamePass.php

```diff
@@ -171,8 +175,12 @@ protected function ensureClassExists($name, $stmt)
     /**
      * Ensure that a referenced interface exists.
      *
-     * @param string $name
-     * @param Stmt   $stmt
+     * @param $interfaces
+     * @param Stmt $stmt
+     *
+     * @throws FatalErrorException
+     *
+     * @internal param string $name
```

Comment:
[I added a `/** @var */` annotation for `$name`, just in case :)](https://github.com/bobthecow/psysh/blob/develop/src/Psy/CodeCleaner/ValidClassNamePass.php#L186)